torch/package/_file_structure_representation.py

```diff
@@ -24,6 +24,17 @@ def add_file(self, file_path):
         folder = self.get_folder(folders)
         folder.children[file] = Folder(file, False)
 
+    def has_file(self, filename: str):
+        lineage = filename.split("/", maxsplit=1)
```

Comment:
Consider using `os.sep` instead of `/` for portability (is that important to us? I'm not sure...). Consider also using `os.path.join/split` but they may not be able to do what you need here.